apps/address-service/domains/common/utils/services/search/searchServiceUtils.js

```diff
@@ -6,6 +6,33 @@ const { md5 } = require('@condo/domains/common/utils/crypto')
 
 const ADDRESS_ITEM_FIELDS = 'id address key meta overrides'
 
+async function upsertAddressSource (context, addressSourceServerUtils, dvSender, normalizedSource, addressId) {
+    if (!normalizedSource) return null
```

Comment:
You search AddressSource by lower-cased source, but save/update without lowercasing.
```suggestion
    if (!normalizedSource) return null
    
    const source = normalizedSource.toLowerCase()
```